frontend/src/views/WheresMyGeneV2/components/HeatMap/index.tsx

```diff
@@ -185,55 +184,62 @@ export default memo(function HeatMap({
     return result;
   }, [selectedGeneExpressionSummariesByTissueName, geneNameToIndex]);
 
-  const initialDisplayedCellTypes = useMemo(() => {
-    return Object.entries(sortedCellTypesByTissueName).reduce(
-      (acc, [tissue]) => {
+  const initialDisplayedCellTypes = useMemo(
+    () =>
+      Object.entries(tissuesByName).reduce((acc, [tissue]) => {
         acc.add(tissue + tissue);
         return acc;
-      },
-      new Set<string>()
-    );
-  }, [sortedCellTypesByTissueName]);
-  // (seve): This state should be moved to the store
-  const [displayedCellTypes, setDisplayedCellTypes] = useState<Set<string>>(
-    initialDisplayedCellTypes
+      }, new Set<string>()),
+    [tissuesByName]
   );
 
   // set of tissue names that are visible and set of cell types that are visible
   // presence is visible
+  const [displayedCellTypes, setDisplayedCellTypes] = useState<Set<string>>(
+    initialDisplayedCellTypes
+  );
 
   useEffect(() => {
     setDisplayedCellTypes(initialDisplayedCellTypes);
-  }, [initialDisplayedCellTypes]);
-
-  const handleExpand = useCallback(() => {
-    const newDisplayedCellTypes = new Set<string>(displayedCellTypes);
-
-    if (expandedTissues.includes("lung")) {
-      setExpandedTissues([]);
-      setDisplayedCellTypes(initialDisplayedCellTypes);
-
-      return;
-    }
-    //expand the first tissue
-
-    sortedCellTypesByTissueName["lung"].forEach((cellType) => {
-      newDisplayedCellTypes.add("lung" + cellType.name);
-    });
-    setExpandedTissues(["lung"]);
-    setDisplayedCellTypes(newDisplayedCellTypes);
-  }, [
-    displayedCellTypes,
-    expandedTissues,
-    sortedCellTypesByTissueName,
-    setExpandedTissues,
-    initialDisplayedCellTypes,
-  ]);
+    setExpandedTissues(new Set<string>());
```

Comment:
This `useEffect` is really just for safety, but this is in case that the list of tissues changes for some reason. If displayedCellTypes is set back to the initial value, and we don't reset the expandedTIssues, it will lead to collapsed tissues with the expanded icon. LMK if you'd like me to create a video to show you this.